rero_ils/facets.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+# Copyright (C) 2020 UCLOUVAIN
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Facets and factories for result aggregation.
+
+See :data:`invenio_records_rest.config.RECORDS_REST_FACETS` for more
+information on how to specify aggregations.
+"""
+
+from __future__ import absolute_import, print_function
+
+from flask import current_app
+
+from .utils import get_current_language
+
+
+def _i18n_aggregations(search, definitions):
+    """Add aggregations to search query.
+
+    :param search: Search object.
+    :param definitions: facet definitions.
+    :returns: the search object.
+    """
+    if definitions:
+        language = get_current_language()
```

Comment:
this should not work as the backend API does not have cookies and session. It would be better to add a new language optional URL parameter.

Reply:
It's done